### ISSUE TITLE
nav label isn't displaying in AAT

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-visua11y#109]
### Subject of the issue
No label is displayed for Visua11y nav button.

![missing-label](https://github.com/cgkineo/adapt-visua11y/assets/7045330/bd6c8d3b-bf85-4ac8-a7d2-4bf255da0e2c)

### Your environment
* v5.35.7 FW / v0.11.2 AAT

### Expected behaviour
There should be a button label 'Accessibility' as per the `navLabel` property default, see [example.json](https://github.com/cgkineo/adapt-visua11y/blob/071bff97b43b2fd6c6a63189cdd5d11f2614dcc8/example.json#L5).

`navLabel` exists in [course.schema.json](https://github.com/cgkineo/adapt-visua11y/blob/071bff97b43b2fd6c6a63189cdd5d11f2614dcc8/schema/course.schema.json#L43) but missing in [properties.schema](https://github.com/cgkineo/adapt-visua11y/blob/master/properties.schema).
